### ISSUE TITLE
a11y: profile page has text in divs

### ISSUE THREAD

**Issue by Danielle254** [LetsGetTechnical/elecretanta#526]
"Interests & Hobbies" badges (the chips that display below this header - for example: "Music & Sound") have text contained inside `<div>` elements. These should be contained within text elements. Recommend `<p>`

![Image](https://github.com/user-attachments/assets/b65bfc5a-cce6-43a3-a4c2-b59ac9a7ee67)
